semcore/base-trigger/CHANGELOG.md

```diff
@@ -2,6 +2,18 @@
 
 CHANGELOG.md standards are inspired by [keepachangelog.com](https://keepachangelog.com/en/1.0.0/).
 
+## [4.49.2] - 2024-12-05
+
+### Removed
+
+- Default tag `button` from FilterTrigger and ButtonTrigger to reduce code duplication.
+
+## [4.49.1] - 2024-11-29
+
+### Changed
+
+- Default tag for BaseTrigger to `button` as part of a11y improvement.
```

Comment:
I see a 'reviewdog' error
```
DevDocs.Names] Use 'A11y' or 'A11Y' instead of 'a11y'.

Raw Output:
{"message": "[DevDocs.Names] Use 'A11y' or 'A11Y' instead of 'a11y'.", "location": {"path": "semcore/base-trigger/CHANGELOG.md", "range": {"start": {"line": 15, "column": 54}}}, "severity": "ERROR"}
```

Maybe it's better to fix it
@ksimanov @ilyabrower what do you think?

Reply:
We write everywhere `a11y`, I think we can leave it as is